etc/docker/Dockerfile

```diff
@@ -57,7 +57,7 @@ ENV OTBR_NORELEASE_DEPS \
 
 RUN apt-get update \
   && apt-get install -y $OTBR_DOCKER_REQS $OTBR_DOCKER_DEPS \
-  && git reset --hard && git clean -xffd \
+  && git clean -xffd \
```

Comment:
```suggestion
  && git clean -xfd \
```
Why `ff`?

Reply:
it will also clean untracked files in sub modules.